### DISCUSSION TITLE
Cloud syncing logic - fixes?

### DISCUSSION THREAD

**Discussion by kailielexx** [hoangsonww/The-MovieVerse-Database#21] [Q&A] [answered]
There seems to be a bug - when the user has signed out, the watchlists of that user remains. Is this a feature or a bug?

**Comment by hoangsonww** [comment, marked as answer]
I do not think this is a bug. My logic is this: when the user has signed in, their email is stored in local storage. It is only updated when another user signs in. This aligns with my logic - I want to allow the app to automatically update and sync their newly created favorites and watchlists even if they are signed out. (So when they sign in again their created watchlist will be updated.) When a new user signs in, these will go away immediately, ensuring that these users' watchlists and favorites do not overlap (i.e. one cannot see another's watchlists and favorites.) Hope this helps!
